src/main/util/consts.py

```diff
@@ -91,10 +91,14 @@ class TASK(Enum):
     MAX_DIGIT = 'max_digit'
 
     @classmethod
-    def tasks(cls):
+    def tasks_values(cls):
         return [cls.PIES.value, cls.MAX_3.value, cls.ZERO.value,
                 cls.ELECTION.value, cls.BRACKETS.value, cls.MAX_DIGIT.value]
 
+    @classmethod
+    def tasks(cls):
+        return [cls.PIES, cls.MAX_3, cls.ZERO, cls.ELECTION, cls.BRACKETS, cls.MAX_DIGIT]
```

Comment:
I think it is better to use the loop: `[task for task in TASK]`

Reply:
maybe it's better to use TASK.__members__.items() here also?

Reply:
`TASK.members` returns dictionary and each key has string type in this case